src/App.tsx

```diff
@@ -15,36 +22,114 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+function getPrepearedGoods(
+  goods: string[],
+  options: { sortField: SortType | ''; isReversed: boolean },
+): string[] {
+  const prepearedSortGoods = [...goods];
+
+  if (options.sortField) {
+    prepearedSortGoods.sort((good1, good2) => {
+      switch (options.sortField) {
+        case SortType.Alphabet:
+          return good1.localeCompare(good2);
+
+        case SortType.Length:
+          return good1.length - good2.length;
+
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (options.isReversed) {
+    prepearedSortGoods.reverse();
+  }
+
+  return prepearedSortGoods;
+}
```

Comment:
The function name `getPrepearedGoods` contains a typo. It should be `getPreparedGoods` for clarity and correctness.